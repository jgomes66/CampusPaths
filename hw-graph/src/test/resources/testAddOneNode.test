# Test Adding Just One Node
# Testing to see if one node can be added to a newly created graph

# Create a graph
CreateGraph newGraph

# Add one node to graph
AddNode newGraph firstNode

# Check if graph contains node
ListNodes newGraph