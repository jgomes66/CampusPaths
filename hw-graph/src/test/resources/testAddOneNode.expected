# Test Adding Just One Node
# Testing to see if one node can be added to a newly created graph

# Create a graph
created graph newGraph

# Add one node to graph
added node firstNode to graph newGraph

# Check if graph contains node
newGraph contains: firstNode